tests/models/flan_t5/test_flan_t5.py

```diff
@@ -0,0 +1,18 @@
+# Reference: https://huggingface.co/docs/transformers/en/model_doc/flan-t5
+
+from transformers import AutoModelForSeq2SeqLM, AutoTokenizer
+import pytest
+
+
+@pytest.mark.compilation_xfail
+def test_flan_t5(record_property):
+    record_property("model_name", "FLAN-T5")
+
+    tokenizer = AutoTokenizer.from_pretrained("google/flan-t5-small")
+    model = AutoModelForSeq2SeqLM.from_pretrained("google/flan-t5-small")
+
+    inputs = tokenizer("A step by step recipe to make bolognese pasta:", return_tensors="pt")
+    outputs = model.generate(**inputs)
```

Comment:
as @kevinwuTT mentioned, we think generate is not supported by compiler right now.